docs/UserGuide.adoc

```diff
@@ -14,7 +14,13 @@ endif::[]
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
-== Quick Start
+== Welcome to CYNC
+CYNC is an open-source software that helps small businesses to track and nurture relationships with customers. +
+Derived from the word " Sync ", CYNC aims to keep customers and businesses in sync. CYNC ensures that customers are businesses' main priority
+by storing customers' information and tracking their activities. CYNC also focuses on efficiency thus its
```

Comment:
>Storing customers' information and tracking their activities

Sounds a little sketchy. I think better phrased as something like 'Makes it easy for the business to manage their customers' profile'